commit 880ee765f5738b71f82be586a2e97fb30e420699 (refs/changes/06/3506/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-02-27T15:38:28-08:00 (1 year, 7 months ago)
    
    TRITON-189 change in TRITON-184 broke config-agent running in zones

diff --git a/agent.js b/agent.js
index 9f94266..f07a1fe 100644
--- a/agent.js
+++ b/agent.js
@@ -135,7 +135,8 @@ async.waterfall([
 				// Allow the config file to specify the
 				// instance UUID(s). This is used for load
 				// testing.
-				if (config.instances.length === 0) {
+				if (!config.instances ||
+				    config.instances.length === 0) {
 					config.instances = [ zonename ];
 				}
 			}
